external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopHeroesIT.java

```diff
@@ -45,7 +45,11 @@ public class OpenShiftWorkshopHeroesIT {
     private static final int POSTGRESQL_PORT = 5432;
 
     @Container(image = "${postgresql.13.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
-    static PostgresqlService database = new PostgresqlService();
+    static PostgresqlService database = new PostgresqlService()
+            //fixme https://github.com/quarkus-qe/quarkus-test-framework/issues/455
```

Comment:
TODO has IDE highlight, not sure if fixme does

Reply:
It does, at least in idea